tailwind.config.js

```diff
@@ -1,24 +1,28 @@
+const colors = require('tailwindcss/colors')
+
 module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        'sans': 'Proxima Nova, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
+        sans:
+          'Proxima Nova, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"',
```

Comment:
I'm guessing prettier put this on a second line due to line length - wondering if we can disable that.